docs/UserGuide.md

```diff
@@ -592,14 +610,14 @@ The current time marker (denoted by the red line on the right side of the calend
 | **find**         | `find [n/START_OF_NAME]... [a/PART_OF_ADDRESS]... [pri/PRIORITY]... [income/INCOME]...`<br> e.g.,`find n/A n/B a/clementi pri/High`                                                                 |
 | **sort**         | `sort [name] [address] [priority] [income] [updated]`<br> e.g.,`sort name`                                                                                                                          |
-| **delete**       | `delete INDEXES`<br> Indexes are the index of the person in the person list <br>e.g., `delete 2,3`                                                                                                  |
-| **get**          | `get PARAMETERS`<br> e.g.,`get email phone`                                                                                                                                                         |
+| **delete**       | `delete INDEXES`<br> e.g., `delete 2,3`                                                                                                                                                             |
+| **get**          | `get [name] [phone] [email] [address]`<br> e.g.,`get email phone`                                                                                                                                   |
 | **clear**        | `clear`                                                                                                                                                                                             |
 | **addappt**      | `addappt INDEX date/yyyy-MM-dd from/HH:mm to/HH:mm`<br> e.g., `addappt 1 date/2024-11-08 from/16:00 to/17:00`                                                                                       |
 | **listappt**     | `listappt`                                                                                                                                                                                          |
 | **editappt**     | `editappt INDEX [date/yyyy-MM-dd] [from/HH:mm] [to/HH:mm]`<br> e.g., `editappt 2 from/10:00 to/11:30`                                                                                               |
 | **deleteappt**   | `deleteappt INDEX`                                                                                                                                                                                  |
-| **statistics**   | `statistics` <br> Shows general statistics                                                                                                                                                          |
+| **statistics**   | `statistics` <br> e.g., `statistics`                                                                                                                                                                |
```

Comment:
The example does not feel needed as the other commands that does not take commands don't have examples.